jdbc/src/main/java/org/apache/iotdb/jdbc/IoTDBSQLException.java

```diff
@@ -23,17 +23,17 @@
 public class IoTDBSQLException extends SQLException {
 
   private static final long serialVersionUID = -3306001287342258977L;
-  private String errorMessage;
+//  private String errorMessage;
 
   public IoTDBSQLException(String reason) {
     super(reason);
   }
 
-  public String getErrorMessage() {
-    return errorMessage;
-  }
-
-  public void setErrorMessage(String errorMessage) {
-    this.errorMessage = errorMessage;
-  }
+//  public String getErrorMessage() {
+//    return errorMessage;
+//  }
+//
+//  public void setErrorMessage(String errorMessage) {
+//    this.errorMessage = errorMessage;
+//  }
```

Comment:
Why not just remove these code？

Reply:
+1, is it for sure that these codes will no longer be used?

Reply:
removed